Frontend/src/components/Navbar.vue

```diff
@@ -3,7 +3,8 @@
     <v-menu  
         bottom 
         offset-y
-        transition="slide-x-transition">
+        transition="slide-x-transition"
+        v-if="stored.state.isLogin">
```

Comment:
Why this change?

Reply:
Privacy policy should be accessible by anyone, only logged in users should see our features like map view and point editor

Reply:
Coolio